projects/angular/src/emphasis/alert/_alert.clarity.scss

```diff
@@ -175,20 +196,15 @@
   }
 
   .alert-icon-wrapper {
-    flex: 0 0 $alertIconWidth;
-    align-self: start;
-    padding-top: variables.$clr_baselineRem_1px;
-    @include mixins.css-var(
-      height,
-      clr-alert-icon-size,
-      variables.$clr_baselineRem_0_75,
-      variables.$clr-use-custom-properties
-    );
+    flex-grow: 0;
+    flex-shrink: 0;
+    @include mixins.css-var(flex-basis, clr-alert-icon-size, $alertIconWidth, variables.$clr-use-custom-properties);
   }
 
   .alert-item {
     flex: 1 1 auto;
     display: flex;
+    align-items: center;
```

Comment:
`align-items: center` is true for text only. Icons are top aligned. Check multi line  alerts.